src/controllers/routes.js

```diff
@@ -70,6 +71,30 @@ router.post("/create_playlist", middleware.verifyToken, (req, res) => {
 	}
 });
 
+router.get("/playlist_details", middleware.verifyToken, (req, res) => {
+	if (req.noRefreshToken) {
+		res.redirect("/login");
+	} else {
+		Playlist.findOne({ refreshToken: req.cookies.refreshToken}).sort({ _id: -1 })
+		.then( results => {
+			console.log(results.playlistId)
```

Comment:
remove `console.log(results.playlistId)`

Reply:
Fixed in [this commit.](https://github.com/numel007/SPG-nodejs/pull/16/commits/977270f9bcf9671ee8abe3f91b6b13f19e8b1281).